glsl/postprocess_chronoshift.frag

```diff
@@ -0,0 +1,31 @@
+#version {VERSION}
+#ifdef GL_ES
+precision mediump float;
+#endif
+
+uniform float Frac;
```

Comment:
could this variable get a better name? Or perhaps an explanation what it stands for?